src/UI/Button.js

```diff
@@ -6,12 +6,14 @@ const StyledButton = styled.button`
   height: ${(props) => props.height};
   width: ${(props) => props.width};
   text-align: center;
-  color: ${(props) => props.color};
+  color: ${(props) => (props.disabled ? "#505050" : props.color)};
```

Comment:
change disabled to disabled color if it exists. alos have a defauls if color is not passed in.